"""czds.utils.exceptions."""


class CustomExceptionError(Exception):
    """Raised when an error is encountered."""

    def __init__(self, value: str) -> None:
        """Raises when the value is unknown."""
        pass
